### ISSUE TITLE
unused_variable: false positive if used in read table

### ISSUE THREAD

**Issue by mbtools** [abaplint/abaplint#1619]
```abap
REPORT zfoobar.
DATA lv_test TYPE string.
DATA lv_result TYPE string.
DATA lt_test TYPE STANDARD TABLE OF string.
READ TABLE lt_test INTO lv_result WITH KEY table_line = lv_test. "<<< used
IF sy-subrc = 0.
  WRITE lv_result.
ENDIF.
```

/zfoobar.prog.abap[2, 6]: Variable "lv_test" not used(unused_variables)

**Comment by larshp** [comment]
try again, 2.63.2

**Comment by mbtools** [comment]
👍   this made a big difference.
